### DISCUSSION TITLE
Using a new skeleton for labeled frames

### DISCUSSION THREAD

**Discussion by IsabelleSajonia** [talmolab/sleap#1617] [Help!]
Hi. I currently have a lot of labeled frames but I'm curious if there's a way I can train a separate model (new project) re-using the labeling from the previous project. I want to use mostly the same skeleton as my last one but without tracking the tail points. It looks like if I delete the tail points from the skeleton it will delete them from all labeled frames. 

Is there a way to import labels and re-train without the tail points?

**Comment by talmo** [comment]
Hi @IsabelleSajonia,

Sure thing. So the steps to do this are:

1. Open the old project (with the tail points).
2. File --> Save as to a new `.slp` file.
3. Delete the tail nodes from the Skeleton panel.
4. Continue labeling as normal, add new videos, etc.

If you already started another project without tail nodes, you should be able to import it as well and there will be a dialog window that pops up to resolve potential conflicts.

Let us know if you run into any issues!

Talmo

**Reply by IsabelleSajonia** [reply]
That worked for me thank you! For this model without a tail, I need to start training a new model I'm guessing? I tried resuming training from the previous model and it didn't seem to work.

update: I was able to get a prediction after running a new model and the tail is correctly not labeled, however when I click on the prediction I still see the tail points but visibility is toggled. Will these show up as missing points from the skeleton in a behavior classifier (I use simba)? I'm just not sure why I can still see them. Here's what it looks like: 
![image](https://github.com/talmolab/sleap/assets/70863857/e6fed79e-1075-4baa-b6bf-c51b7bfa8a01)

**Reply by talmo** [reply]
Hi @IsabelleSajonia,

Yep, you need to train a new model. When you did that, did you select one of the previous model's `training_config.json`? If so, it's possible it retained the other nodes. The way to avoid that is to just configure the training job from the defaults rather than loading the settings from a previous config.

Right now it does treat them as not visible, but this could be for a few different reasons (including maybe a bug). After you deleted the nodes from the skeleton, did you still see the invisible ones or did they only appear _after_ you trained and got predictions back?

Cheers,

Talmo
